cyral/resource_cyral_repository.go

```diff
@@ -48,6 +49,13 @@ func resourceRepository() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"labels": {
+				Type:     schema.TypeList,
```

Comment:
We had some issues with list types in schema regarding consecutive `terraform apply` operations. I don't recall the whole scenario at the top of my head, but @VictorGFM would for sure know what to do and how to make sure test-wise that this `list` is good or not to go in this case.

Reply:
Thanks fot the comments, I think that they are not an ordered collection.
Can you confirm this for me @day1118 ?

Reply:
@zero101010 I think TypeSet is more adequate for the repo labels in this case, please change from TypeList to TypeSet, thanks.